### DISCUSSION TITLE
Why is Cypress promoting Playwright?

### DISCUSSION THREAD

**Discussion by resuarez** [cypress-io/cypress#26754] [General]
I don't know if it is the good place to report this.  But, I've checked everywhere and I couldn't find any email to do it.
I've been looking information to run Cypress in docker and I've found this blog in cypress.io:
https://www.cypress.io/blog/2019/05/02/run-cypress-with-a-single-docker-command/
There, it is a link to another blog, [“End-to-End Testing Web Apps: The Painless Way”](https://mtlynch.io/painless-web-app-testing/?ref=cypress.io) by [Michael Lynch](https://twitter.com/deliberatecoder?ref=cypress.io).
![image](https://github.com/cypress-io/cypress/assets/15803627/5a749a6e-d83c-45a4-b670-e110b4f78a57)

However, in that blog, he says that he recommends Playwright instead:
![image](https://github.com/cypress-io/cypress/assets/15803627/332acd27-7273-4fae-9339-4c7879d797e9)

Now, I don't know if I should follow the advice and check playwright instead.

**Comment by emilyrohrbough** [comment]
@resuarez Thank you for reading our blog! This article is quite an old blog post (2019). Michael, the author of the article you linked to [“End-to-End Testing Web Apps: The Painless Way”](https://mtlynch.io/painless-web-app-testing/?ref=cypress.io) (2021) is not a representative of Cypress, just a previous Cypress users who had a solid understanding of the product. I reviewed the content you linked, and it seems like one of Michael’s primary reasons for moving on to Playwright was the fact he did not leverage the Cypress GUI which enabled development. Of course Cypress would love everyone to use Cypress, however, if Playwright is better suited for his developer needs that is okay.

We updated this blog to remove the article link to remove any confusion that Cypress is promoting Playwright over itself.
